### PR TITLE
feat(fee): add instant trade buyer to accept general outputs

### PR DESCRIPTION
**Pull request by JoshuaC817**
#### What this PR does / why we need it:
Adding a general `outputs` field to instant trade buyer builder to cater for any upcoming fees

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors 
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

**Comment by JoshuaC817** [comment]
- [ ] Fixing output indexes

**Comment by JoshuaC817** [comment]
Some success sample txn

1. Buy inscription without royalty
https://mempool.space/testnet/tx/24b72f55eaa7fb36834d068fd28e119c5eaa090f259fdb95206767d1ae00163a
Screenshot 2024-03-08 at 10 44 12 PM

Screenshot 2024-03-08 at 11 18 30 PM


2. Buy inscription with royalty
https://mempool.space/testnet/tx/0fd3968069edcc60cb085ca09f372198ca0f6a5f453226fa595bafc858cefac4
Screenshot 2024-03-08 at 10 56 30 PM

Screenshot 2024-03-08 at 11 25 57 PM

File: packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts [hunk 1]
```diff
@@ -90,6 +92,12 @@ export default class InstantTradeBuyerTxBuilder extends InstantTradeBuilder {
     })
   }
 
+  private bindIncomingOutputs() {
+    if (!!this.outs) {
+      this.outputs = [...this.outputs, ...this.outs]
```
[the remark this answers is not in the document]
Comment (devlancer412) [reply packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts:97]: Could you send me your contact?

File: packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts [hunk 2]
```diff
@@ -24,18 +26,20 @@ export default class InstantTradeBuyerTxBuilder extends InstantTradeBuilder {
...
       datasource,
       network,
       publicKey,
-      feeRate
+      feeRate,
     })
 
     this.receiveAddress = receiveAddress
     this.decodeSellerPSBT(sellerPSBT)
+    this.outs = outputs ?? []
```
Comment (kevzzsk) [inline packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts:42]: let's rename `this.outs` to be consistent with the function naming so we can call this incoming outputs as well. Can keep the params above as `outputs` to be consistent with inscriber class

Comment (JoshuaC817) [reply packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts]: updated `this.outs` to `this.incomingOutputs`

**Pull request merged**